api/src/main/java/org/openmrs/module/aijar/metadata/concept/Dictionary.java

```diff
@@ -0,0 +1,157 @@
+package org.openmrs.module.aijar.metadata.concept;
```

Comment:
@ssmusoke, I lifted the whole file off KenyaEMR, it has some pretty useful methods, and I am using the getConcept() method in a number of places